source/_posts/2020-06-03-release-111.markdown

```diff
@@ -838,6 +841,7 @@ The integrations below have been removed:
 [@bdraco]: https://github.com/bdraco
 [@bieniu]: https://github.com/bieniu
 [@boralyl]: https://github.com/boralyl
+[@bouwew]: https://github.com/bouwew
```

Comment:
This is generated. Please move it to the same paragraph (locally within the paragraph) as it is used (at the end), as I mentioned in my initial comment.

Reply:
Quickly installed Jekyll to ensure it was rendering ok, I hope it's correct now (it renders correctly locally). Apologies and will fetch more coffee next commit round.